region/brinstar/pink/Mission Impossible Room.json

```
@@ -1028,7 +1028,7 @@
     {
       "id": 44,
       "link": [4, 4],
-      "name": "R-Mode Pause Abuse Spark Interrupt (Gain Blue Suit) (Unbroken Super Block)",
+      "name": "Unbroken Super Block, R-Mode Pause Abuse Spark Interrupt",
```

This is not important enough to be in the title, but should be added to the note instead.
```suggestion
      "name": "R-Mode Pause Abuse Spark Interrupt",
```